server/src/Infrastructure/Persistence/Configurations/FlashcardSetTypeConfiguration.cs

```diff
@@ -8,44 +8,13 @@ public class FlashcardSetEntityTypeConfiguration : IEntityTypeConfiguration<Flas
 {
     public void Configure(EntityTypeBuilder<FlashcardSet> builder)
     {
-        builder.HasKey(f => f.Id);
-
-        builder.Property(f => f.Id)
-            .ValueGeneratedOnAdd()
-            .HasColumnName("id");
-
         builder.HasOne<Lecture>()
             .WithMany()
-            .HasForeignKey(f => f.LectureId)
-            .HasConstraintName("lecture_id")
+            .HasForeignKey(fs => fs.LectureId)
             .OnDelete(DeleteBehavior.SetNull);
 
-        builder.Property(f => f.Name)
-            .IsRequired()
-            .HasColumnType("VARCHAR")
-            .HasMaxLength(50)
-            .HasColumnName("name");
-
-        builder.Property(f => f.CreatedBy.User)
-            .IsRequired()
-            .HasColumnType("VARCHAR")
-            .HasMaxLength(50)
-            .HasColumnName("createdby_user");
-
-        builder.Property(f => f.CreatedBy.Timestamp)
-            .IsRequired()
-            .HasColumnType("DATETIME2")
-            .HasColumnName("createdby_timestamp");
-
-        builder.Property(f => f.UpdatedBy.User)
-            .IsRequired()
-            .HasColumnType("VARCHAR")
-            .HasMaxLength(50)
-            .HasColumnName("updatedby_user");
+        builder.OwnsOne(e => e.CreatedBy);
```

Comment:
Mark the Stamp as Owned

https://learn.microsoft.com/en-us/ef/core/modeling/owned-entities